### PR TITLE
PUBDEV-4190: Arguments in h2o.naiveBayes in R are not the same as Python/Java

### PR DESCRIPTION
**Pull request by navdeep-G**

### CODE REVIEW

File: h2o-bindings/bin/gen_R.py [hunk 1]
```diff
@@ -40,13 +40,20 @@ def gen_module(schema, algo, module):
             continue
         if algo == "naivebayes":
             if param["name"] == "min_sdev":
-                yield "#' @param threshold The minimum standard deviation to use for observations without enough data. "
+                yield "#' @param threshold This argument is deprecated, use `min_sdev` instead. The minimum standard deviation to use for observations without enough data. "
+                yield "#'                  Must be at least 1e-10."
```
Comment (angela0xdata) [inline h2o-bindings/bin/gen_R.py]: The following test shows a min_sdev value of -60. Should that be updated?
https://github.com/h2oai/h2o-3/blob/master/h2o-r/tests/testdir_algos/naivebayes/runit_naivebayes_grid_cars_negative.R

File: h2o-bindings/bin/gen_R.py [hunk 2]
```diff
@@ -40,13 +40,20 @@ def gen_module(schema, algo, module):
...
+                yield "#' @param min_sdev The minimum standard deviation to use for observations without enough data. "
                 yield "#'                  Must be at least 1e-10."
                 continue
             if param["name"] == "eps_sdev":
-                yield "#' @param eps A threshold cutoff to deal with numeric instability, must be positive."
+                yield "#' @param eps This argument is deprecated, use `eps_sdev` instead. A threshold cutoff to deal with numeric instability, must be positive."
+                yield "#' @param eps_sdev A threshold cutoff to deal with numeric instability, must be positive."
```
Comment (angela0xdata) [inline h2o-bindings/bin/gen_R.py]: The following test shows an eps_sdev value of -60. Should that be updated?
https://github.com/h2oai/h2o-3/blob/master/h2o-r/tests/testdir_algos/naivebayes/runit_naivebayes_grid_cars_negative.R

File: h2o-bindings/bin/gen_R.py (hunk 1 above)
Comment (navdeep-G) [inline h2o-bindings/bin/gen_R.py]: Those tests are for invalid parameters.

File: h2o-bindings/bin/gen_R.py (hunk 2 above)
Comment (navdeep-G) [inline h2o-bindings/bin/gen_R.py]: Those tests are for invalid parameters.

File: h2o-bindings/bin/gen_R.py (hunk 1 above)
Comment (angela0xdata) [inline h2o-bindings/bin/gen_R.py]: Hmm... Those were the only tests in R that I could find that use those parameters. Do we need a test for the valid parameters?

Comment (navdeep-G) [inline h2o-bindings/bin/gen_R.py]: Ah, those parameters are not there because they are introduced in this PR.

Comment (navdeep-G) [inline h2o-bindings/bin/gen_R.py]: See here:

**Pull request merged**